scripts/c-to-cpp-includes.pl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
+
+# sweep over the squid repository, looking for patterns of
+# c standard library header files that have a c++ equivalent,
+# replacing these includes with the sequivalent standard c++ header.
+
+
+#header files to consider (without the .h extension)
+@headers=qw(
+assert
+ctype
+errno
+inttypes
+limits
+math
+signal
+stdarg
+stddef
+stdint
+stdio
+stdlib
+string
+time
+unistd
```

Comment:
There is no such header as `cunistd`
```suggestion
```

Reply:
Removed. I'll have to undo and redo the changes in the codebase